kyaml/openapi/README.md

```diff
@@ -67,11 +67,25 @@ make kubernetesapi/swagger.go API_VERSION=v1.21.2
 While the above commands generate the swagger.go files, they
 do not make them available for use nor do they update the
 info field reported by `kustomize openapi info`. To make the
-newly fetched schema and swagger.go available:
+newly fetched schema and swagger.go available, update the
+file [`kubernetesapi/openapiinfo.go`](https://github.com/kubernetes-sigs/kustomize/blob/master/kyaml/openapi/kubernetesapi/openapiinfo.go). 
+
+Here is an example of what it looks like with v1.21.2. The version number needs to be updated in all five places that it appears:
```

Comment:
> This section is describing how to add additional schemas other than the default, which the example doesn't show but the old script handled. It's a shame to lose that.

I can add instructions for how to add schemas other than the default, and how to change the default. This should maintain what the old script handles. 

> We're removing the periods, making it impossible to parse accurately. But why do we do that? Could we replace them with underscores instead and use that as the source of the version in the openapiinfo.go generation script?

I tried replacing the dots with underscores. It looks like the swagger.go file still has a `package v1212` at the top (no underscores). I agree that it's not possible to parse completely accurately, but is it likely that we will ever have a case where it is ambiguous? As it is, kubectl doesn't seem to like it when we have multiple versions, so I'm doubtful that we'll ever run into a case where we have multiple versions that look the same when we remove the dots.

Reply:
> We're removing the periods, making it impossible to parse accurately. But why do we do that? Could we replace them with underscores instead and use that as the source of the version in the openapiinfo.go generation script?
>
>> Done

I see you've updated them to use underscores, but the openapiinfo.go generation script is still gone. Did you try updating it to use the directory version information instead of the JSON and it didn't work?

Reply:
> Did you try updating it to use the directory version information instead of the JSON and it didn't work?

Just did this! For some reason I missed that part of your comment. It worked well, thank you for the suggestion.